src/charts/chartOptions.js

```diff
@@ -6,22 +6,22 @@ define(['jquery', 'common/rivetsExtra', "charts/chartWindow", "charts/charts", '
 
     var state = [], view = [], template_manager = {}, stringWidth= {}, isListenerAdded = false;
 
-    var timeperiod_arr = [{value: "1t", name: "1 Tick", digit: 1, type: "ticks"}, 
```

Comment:
@arnabk if it's any help, github provides a parameter to ignore whitespace changes; https://github.com/binary-com/webtrader/pull/624/files?w=1